### PR TITLE
change nongo bundle generation cruft to use updated flags

### LINKED ISSUE #5987: after init of the project Makefile doesn't have proper bundle target
#### What did you do?

after initializing helm based operator project by command
```
operator-sdk init --plugins helm --domain $(DOMAIN) --group $(GROUP) --version $(APIVERSION) --kind $(KIND) --helm-chart=../charts/$@/ --project-name prefix-$@
```
Makefile doesn't have proper bundle target with right option.
current code - https://github.com/operator-framework/operator-sdk/blob/025df62736f9e98ad22af1384f1d67d96cbcb7ee/internal/plugins/manifests/v2/init.go#L185

Which should be:
```diff
-	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle $(BUNDLE_GEN_FLAGS)
```
Like on the line https://github.com/operator-framework/operator-sdk/blob/025df62736f9e98ad22af1384f1d67d96cbcb7ee/internal/plugins/manifests/v2/init.go#L176

#### What did you expect to see?

digests injected to the manifests properly.

#### What did you see instead? Under which circumstances?

image tags are used instead of digests

#### Environment

**Operator type:**
/language helm

**Kubernetes cluster type:**

OpenShift

`$ operator-sdk version`

operator-sdk version: "v1.22.2", commit: "da3346113a8a75e11225f586482934000504a60f", kubernetes version: "1.24.1", go version: "go1.18.4", GOOS: "darwin", GOARCH: "amd64"

`$ kubectl version`

Client Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.8-dispatcher", GitCommit:"283881f025da4f5b3cefb6cd4c35f2ee4c2a79b8", GitTreeState:"clean", BuildDate:"2021-09-14T05:14:54Z", GoVersion:"go1.15.13", Compiler:"gc", Platform:"darwin/amd64"}

#### Possible Solution

Update current code - https://github.com/operator-framework/operator-sdk/blob/025df62736f9e98ad22af1384f1d67d96cbcb7ee/internal/plugins/manifests/v2/init.go#L185

to following:
```diff
-	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle $(BUNDLE_GEN_FLAGS)
```

#### Additional context

N/A

### PR DESCRIPTION
**Pull request by jberkhahn**
Fixes #5987

Took a look at this and it looks like it should be fine.

### CODE REVIEW

**Comment by jberkhahn** [comment]
hey could you take a look at this @jmrodri ? You wrote the original change for the go stuff

**Review by everettraven — APPROVED** [review]
Looks good to me, but because I am unfamiliar with the reasoning for only having this done in the Go plugin in the past I will leave a LGTM up to @jmrodri or others that are familiar with it.

File: internal/plugins/manifests/v2/init.go [hunk 1]
```diff
@@ -184,7 +184,7 @@ bundle: manifests kustomize ## Generate bundle manifests and metadata, then vali
 bundle: kustomize ## Generate bundle manifests and metadata, then validate generated files.
 	operator-sdk generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
-	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle $(BUNDLE_GEN_FLAGS)
```
Comment (varshaprasad96) [inline internal/plugins/manifests/v2/init.go:187]: Can we leave the `--overwrite` flag as is. Removing it will affect the user experience.

**Review by varshaprasad96 — CHANGES_REQUESTED** [review]

**Review by jmrodri — APPROVED** [review]
/lgtm

File: internal/plugins/manifests/v2/init.go (hunk 1 above)
Comment (varshaprasad96) [reply internal/plugins/manifests/v2/init.go:187]: Ah! missed it! Sounds good!

**Review by varshaprasad96 — APPROVED** [review]
/lgtm

**Pull request merged**